SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -223,9 +226,98 @@ void USpatialWorkerConnection::ConnectToLocator()
 	});
 }
 
-bool USpatialWorkerConnection::ShouldConnectWithLocator()
+void USpatialWorkerConnection::ConnectToLocator()
+{
+	if (LocatorConfig.WorkerType.IsEmpty())
+	{
+		LocatorConfig.WorkerType = SpatialConstants::ClientWorkerType;
+	}
+
+	if (LocatorConfig.WorkerId.IsEmpty())
+	{
+		LocatorConfig.WorkerId = LocatorConfig.WorkerType + FGuid::NewGuid().ToString();
+	}
+
+	FTCHARToUTF8 PlayerIdentityTokenCStr(*LocatorConfig.PlayerIdentityToken);
+	FTCHARToUTF8 LoginTokenCStr(*LocatorConfig.LoginToken);
+
+	Worker_Alpha_LocatorParameters LocatorParams = {};
+	LocatorParams.player_identity.player_identity_token = PlayerIdentityTokenCStr.Get();
+	LocatorParams.player_identity.login_token = LoginTokenCStr.Get();
+
+	WorkerLocator = Worker_Alpha_Locator_Create(TCHAR_TO_UTF8(*LocatorConfig.LocatorHost), 0, &LocatorParams);
```

Comment:
What's the `0` parameter? Can you add a comment?

Reply:
I'm not sure what the `0` is for either.  It corresponds to the port parameter, but it's undocumented in the header.  I'm assuming it allows a random port.
```
/**
 * Creates a client which can be used to connect to a SpatialOS deployment via the locator service.
 * This is the new flow used to connect a local worker to a cloud deployment with a player identity
 * token and login token.
 * This object should not be used concurrently by multiple threads.
 */
WORKER_API Worker_Alpha_Locator*
Worker_Alpha_Locator_Create(const char* hostname, uint16_t port,
                            const Worker_Alpha_LocatorParameters* params);
```

Reply:
Followed up with the worker team, `0` means to use the default locator port.  I've added a comment to reflect this.